server/views/metadata.py

```diff
@@ -11,11 +11,11 @@
-COUNTRY_OF_FOCUS_DEFAULTS = [{'label':'United States', 'tags_id': 9361424, 'tag_sets_id': 1970, 'tag_set_name': 'subject_country','tag_set_label':'About Country'}, {'label':'United Kingdom', 'tags_id': 9361436, 'tag_sets_id': 1970, 'tag_set_name': 'subject_country','tag_set_label':'About Country'},{'label': 'India', 'tags_id': 9353533, 'tag_sets_id': 1970, 'tag_set_name': 'subject_country', 'tag_set_label': 'About Country'}, {'label':'Germany', 'tags_id': 9361525, 'tag_sets_id': 1970, 'tag_set_name': 'subject_country','tag_set_label':'About Country'}, {'label': 'Spain', 'tags_id': 9361549, 'tag_sets_id': 1970, 'tag_set_name': 'country_of_focus', 'tag_set_label': 'About Country'},{'label': 'Italy', 'tags_id': 9361490, 'tag_sets_id': 1970, 'tag_set_name': 'subject_country', 'tag_set_label': 'About Country'}, {'label': 'France', 'tags_id': 9361487, 'tag_sets_id': 1970, 'tag_set_name': 'subject_country', 'tag_set_label': 'About Country'}]
-
+PUBLICATION_COUNTRY_DEFAULTS = [{'label':'Kenya', 'tags_id': 5936877, 'tag_sets_id': 15, 'tag_set_name': 'pub_country','tag_set_label':'Publication Country'}, ]
+PUBLICATION_STATE_DEFAULTS = []
+PRIMARY_LANGUAGE_DEFAULTS = [{'label': 'English', 'tags_id': 5890352, 'tag_sets_id': 6, 'tag_set_label':'Primary Language', 'tag_set_name': 'primary_language'}, {'label': 'Swahili', 'tags_id': 5890355, 'tag_sets_id': 6, 'tag_set_label':'Primary Language', 'tag_set_name': 'primary_language'}, {'label': 'French', 'tags_id': 5890354, 'tag_sets_id': 6, 'tag_set_label':'Primary Language', 'tag_set_name': 'primary_language'}, {'label': 'Spanish', 'tags_id': 5890467, 'tag_sets_id': 6, 'tag_set_label':'Primary Language', 'tag_set_name': 'primary_language'}, {'label': 'German', 'tags_id': 5890478, 'tag_sets_id': 6, 'tag_set_label':'Primary Language', 'tag_set_name': 'primary_language'}, {'label': 'Zuru', 'tags_id': 5890390, 'tag_sets_id': 6, 'tag_set_label':'Primary Language', 'tag_set_name': 'primary_language'}]
+COUNTRY_OF_FOCUS_DEFAULTS = [{'label':'Kenya', 'tags_id': 5936877, 'tag_sets_id': 15, 'tag_set_name': 'pub_country','tag_set_label':'Publication Country'}]
+MEDIA_TYPE_DEFAULTS = [{'label': 'Print Native', 'tags_id': 5933332, 'tag_sets_id': 8, 'tag_set_name': 'media_format', 'tag_set_label': 'Media Format'},]
```

Comment:
This will never be used since we now have `media_type` tags in the database but this was just to be used just in case a database didn't have any `media_types`